.github/workflows/notify-triager.yml

```diff
@@ -0,0 +1,56 @@
+name: Notify Triage Maintainers
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, edited, ready_for_review]
+
+jobs:
+  Notify-triagers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4.1.1
+
+      - name: Check PR Changes for .md files
+        id: md-pr-changes
+        uses: tj-actions/changed-files@v42
+        with:
+          files: |
+            **.md
+      - name: Check PR Changes for non-.md files
+        id: non-md-pr-changes
+        uses: tj-actions/changed-files@v42
+        with:
+          files: |
+            !**.md
+  
+      - name: Read triagers.json
+        id: read-triagers-json
+        run: |
+          mdTriagers=$(jq -c '.mdTriagers' triagers.json)
+          codeTriagers=$(jq -c '.codeTriagers' triagers.json)
+          echo "::set-output name=md-triagers::$mdTriagers"
+          echo "::set-output name=code-triagers::$codeTriagers"
+
+      - name: Create Comment for non-.md files
+        if: steps.non-md-pr-changes.outputs.any_changed == 'true'
+        run: |
+          curl \
+          -X POST \
+          -H "Authorization: token ${{ secrets.GH_TOKEN }}" \
+          -H "Accept: application/vnd.github.v3+json" \
+          https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
+          -d '{
+            "reviewers": ${{ steps.read-triagers-json.outputs.code-triagers }}
```

Comment:
If there's already a CODEOWNER assigned for review, would that be replaced with the ones in this array from `triagers.json`?

Reply:
Nope, that will be done by Github automatically.